app/src/main/res/values/strings.xml

```diff
@@ -420,6 +420,7 @@
     <string name="group_name_error_exists">Please choose a name that is unique</string>
     <string name="edit_group">Edit group</string>
     <string name="play_automatically">Play automatically</string>
+    <string name="show_updates">Show app updates</string>
```

Comment:
```suggestion
    <string name="show_updates">Automatically check for new updates</string>
```

Reply:
renamed to check updates automatically as the title was large and hidden in settings